jstests/aggregation/bugs/server9625.js

```diff
@@ -50,7 +50,7 @@ load('jstests/aggregation/extras/utils.js');
     // isAssociativeAndCommutative() is called. If all arguments are constant, the optimization
```

Comment:
While you're in this file, can you update this comment to not refer to `isAssociativeAndCommutative()`, and instead to the two new methods? Also on line 67.